### PR TITLE
docker v19.03.10

### PR DESCRIPTION
**Pull request by greut**
- https://github.com/docker/docker-ce/releases/tag/v19.03.9
- https://github.com/docker/docker-ce/releases/tag/v19.03.10

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -120,40 +120,40 @@
...
-		{"path":"github.com/docker/docker/registry/resumable","checksumSHA1":"jH7uQnDehFQygPP3zLC/mLSqgOk=","revision":"aa6a9891b09cce3d9004121294301a30d45d998d","revisionTime":"2020-01-17T19:55:42Z","version":"v19.03.8","versionExact":"v19.03.8"},
-		{"path":"github.com/docker/docker/rootless","checksumSHA1":"NgEtGryOwSLJ6QRlMwNcDLp1zIM=","revision":"aa6a9891b09cce3d9004121294301a30d45d998d","revisionTime":"2020-01-17T19:55:42Z","version":"v19.03.8","versionExact":"v19.03.8"},
-		{"path":"github.com/docker/docker/volume","checksumSHA1":"Bs344j8rU7oCQyIcIhO9FJyk3ts=","revision":"aa6a9891b09cce3d9004121294301a30d45d998d","revisionTime":"2020-01-17T19:55:42Z","version":"v19.03.8","versionExact":"v19.03.8"},
-		{"path":"github.com/docker/docker/volume/mounts","checksumSHA1":"PNKeHho5s98kfdQAd+3+3442luU=","revision":"aa6a9891b09cce3d9004121294301a30d45d998d","revisionTime":"2020-01-17T19:55:42Z","version":"v19.03.8","versionExact":"v19.03.8"},
+		{"path":"github.com/docker/docker/api/types","checksumSHA1":"mFZrlxHiQuq52BGccWNl/bO1GN0=","revision":"aa6a9891b09cce3d9004121294301a30d45d998d","revisionTime":"2020-01-17T19:55:42Z","version":"v19.03.9","versionExact":"v19.03.8"},
```
Comment (notnoop) [inline vendor/vendor.json:123]: This seems still pointing to the old sha and old versionExact value.

File: vendor/vendor.json [hunk 2]
```diff
@@ -120,40 +120,40 @@
...
+		{"path":"github.com/docker/docker/pkg/pools","checksumSHA1":"dj8atalGWftfM9vdzCsh9YF1Seg=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/stdcopy","checksumSHA1":"w0waeTRJ1sFygI0dZXH6l9E1c60=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/stringid","checksumSHA1":"THVhMDu12TT7TpGJkazOSxQhmRs=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/system","checksumSHA1":"pxRcmuJMrKTcpxDc+xzSVBx3VtM=","comment":"pick up https://github.com/moby/moby/pull/40021","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
```
Comment (notnoop) [inline vendor/vendor.json:148]: Is the comment here still relevant?
```suggestion
		{"path":"github.com/docker/docker/pkg/system","checksumSHA1":"pxRcmuJMrKTcpxDc+xzSVBx3VtM=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
```

Comment (greut) [reply vendor/vendor.json]: probably not, good catch.

**Comment by greut** [comment]
> Looks like we are still having issues with compiling to windows - https://circleci.com/gh/greut/nomad/1273 . It's an issue that was fixed in docker master [moby/moby@c3a0a37](https://github.com/moby/moby/commit/c3a0a3744636069f43197eb18245aaae89f568e5) but haven't been included in [v19.03.9](https://github.com/moby/moby/blob/v19.03.9/pkg/system/filesys_windows.go#L111) (or [v19.03.10 ](https://github.com/moby/moby/blob/v19.03.10/pkg/system/filesys_windows.go#L111), which just got released too) :(.

It's still there, `govendor` permits this kind of cherry-picking. Let's hope things will get nicer before the `go mod` switch can be done.

**Comment by notnoop** [comment]
Sounds good.  Thanks for the update.  Mind if you rebase against master to remove the irrelevant diff caused by merge commit.  I'll merge afterwards.

**Comment by greut** [comment]
https://github.com/docker/docker-ce/releases/tag/v19.03.11 yarg :)

**Pull request merged**